x-pack/plugins/cases/public/components/add_comment/index.tsx

```diff
@@ -177,7 +177,7 @@ export const AddComment = React.memo(
                           data-test-subj="submit-comment"
                           fill
                           iconType="plusInCircle"
-                          isDisabled={isLoading}
+                          isDisabled={!comment || isLoading}
```

Comment:
Should we have similar behaviour in `Edit comment` as well? 
i.e. when` text area` is empty in edit comment, it should disable `save` button and no error message shown?

Reply:
I agree with @js-jankisalvi. We should be consistent.